src/lib/pci_virtio_sock.c

```diff
@@ -652,6 +666,12 @@ static struct pci_vtsock_sock *alloc_sock(struct pci_vtsock_softc *sc)
 	s->local_shutdown = 0;
 	s->peer_shutdown = 0;
 
+	if (0 > asprintf(&s->name, "vsock.%d", id)) {
```

Comment:
`asprintf(...) < 0` is more idiomatic these days.

Up til now I had been using a combination of the src/dst address pairs and the local fd# for identifying debug messages (which is, TBH, rather verbose and not ideal) but how is someone debugging this code supposed map from the id assigned here to an actual flow of interest?